ansible-scylla-node/tasks/common.yml

```diff
@@ -114,23 +114,38 @@
   become: true
   when: skip_ntp is defined and skip_ntp|bool == false
 
-- name: node exporter setup
+- name: check for node_exporter_install
+  stat:
+    path: /usr/sbin/node_exporter_install
+  register: node_exporter_install
+
+- name: setup legacy node exporter
   shell: |
     node_exporter_install --force
-  when: install_type == 'online'
+  when: (install_type == 'online') and (node_exporter_install.stat.exists)
   become: true
   notify:
     - node_exporter start
-  ignore_errors: true
-  #TODO: stop ignoring errors when the node_exporter_install script fix is available in all actual versions, resp. use only for < 5.0 / 2022
 
-- name: node exporter setup from 5.0/2022
```

Comment:
Won't this break support for older versions?

Reply:
nope, tested with both 2021 and 2022, backwards compatible with old node exporter